src/field.cpp

```diff
@@ -1038,6 +1139,25 @@ bool map::process_fields_in_submap( submap *const current_submap,
                         }
 
                         // Consume adjacent fuel / terrain / webs to spread.
+                        // Allow raging fires (and only raging fires) to spread up
+                        // Spreading down is achieved by wrecking the walls/floor and then falling
+                        if( zlevels && cur->getFieldDensity() == 3 && p.z < OVERMAP_HEIGHT ) {
```

Comment:
Am I reading this right? if you burn down a house above a basement you end up with a crater?  If so, BRAVO!